R/truetype.r

```diff
@@ -150,7 +150,13 @@ ttf_find_default_path <- function() {
     return(paths[file.exists(paths)])
 
   } else if (grepl("^mingw", R.version$os)) {
-    return(paste(Sys.getenv("SystemRoot"), "\\Fonts", sep=""))
+    paths <-
+      c(paste(Sys.getenv("SystemRoot"), "\\Fonts", sep=""),
+        file.path(
+          Sys.getenv()["LOCALAPPDATA"],
```

Comment:
I think `Sys.getenv("LOCALAPPDATA")` is a bit cleaner here.

Reply:
Nice! Have no idea `Sys.getenv()` had arguments =) Agree!